wicket/example-rack-update-spec.toml

```diff
@@ -0,0 +1,12 @@
+# This is a specification that is used to generate a rack update  for
+# installation via wicket. The update is generated by the `mupdate-pkg` tool,
+# which generates a `manifest.json` and packages it up along with the files in
+# this spec into a tarball.
+
+version = "1.0.0"
+output_dir = "/tmp"
```

Comment:
This is specifying where the composite tarball gets dumped, right? That doesn't feel like a property of the manifest; should/should it be a command line argument to `mupdate-pkg` instead?